lib/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import { useStyles } from 'sku/react-treat';
+import { Box } from '../Box/Box';
+import { Column } from '../Column/Column';
+import { Columns } from '../Columns/Columns';
+import { IconChevron } from '../icons';
+import { Link, LinkProps } from '../Link/Link';
+import { Overlay } from '../private/Overlay/Overlay';
+import { Text } from '../Text/Text';
+import { Hidden } from '../Hidden/Hidden';
+import { paginate } from './paginate';
+
+import * as styleRefs from './Pagination.treat';
+
+export interface PaginationProps {
+  page: number;
+  total: number;
+  linkProps: (page: number) => LinkProps;
```

Comment:
I think we should make the argument an object with a `page` property to future-proof the API.